src/pages/home/report/ReportActionsView.js

```diff
@@ -397,6 +425,68 @@ class ReportActionsView extends React.Component {
         }
     }
 
+    /**
+     * Scrolls to a specific report action ID
+     */
+    scrollToReportActionID() {
+        this.actionIndexID = _.findIndex(this.sortedReportActions, (
+            ({action}) => parseInt(action.reportActionID, 10) === this.props.reportActionID
+        ));
+
+        if (this.actionIndexID !== -1) {
+            ReportScrollManager.scrollToIndex({index: this.actionIndexID, viewPosition: 0.5});
+            this.setState({selectedReportActionID: this.props.reportActionID});
+        }
+    }
+
+    /**
+     * Records when a report actionID is done rendering.
+     *
+     * @param {string} reportActionID
+     */
+    recordReportActionIDRendered(reportActionID) {
+        this.renderedActionIDs.add(parseInt(reportActionID, 10));
```

Comment:
Would it be better to just wait until _all_ of them are rendered? Seems like something that the `ReportActionsList` could tell us with a callback perhaps.

Reply:
So i tried having a `componentDidUpdate` in `ReportActionsList` but that gets fired a lot earlier than when all items in ReportActionItem finish rendering.
It would be cool to know when all items are rendered but how do you know when thats done? On mobile even it may render just 20 report actions at a time, or just 5 on scroll so how can you find out when all the rendering is done?

Hence because i wasn't able to find a way to know when "all" was done i recorded each over here and if the one we cared about was rendered we could scroll to it. Let me know if you feel there is a way to find out when all is rendered.

Reply:
>So i tried having a componentDidUpdate in ReportActionsList but that gets fired a lot earlier than when all items in ReportActionItem finish rendering.

I was thinking more along the lines of having this logic inside the ReportActionsList and a prop like this

```js
<ReportActionsList
    onAllActionsRendered={() => {//do stuff}}
/>
```

>It would be cool to know when all items are rendered but how do you know when thats done?

Compare the length of actions with the `renderedActionIDs.length`?

>Hence because i wasn't able to find a way to know when "all" was done i recorded each over here and if the one we cared about was rendered we could scroll to it. Let me know if you feel there is a way to find out when all is rendered.

Ok hmm I think I did not understand the problem we are trying to solve then in this case.

Reply:
Maybe we can have the `ReportActionsList` do 

```js
<ReportActionsList
    onActiveReportActionIDRendered={() => { // do stuff }}
/>
```

These suggestions are mostly because I want to try to keep this component lean.

Reply:
Yeah i'll skip it for now but i made a note of it in future improvements in the PR description when we update ReportActionsList and ReportActionsView.